Sejima/Source/MUTime/MUTime.swift

```diff
@@ -0,0 +1,269 @@
+//
+//  MUTime.swift
+//  Sejima
+//
+//  Created by Loïc GRIFFIE on 31/10/2018.
+//  Copyright © 2018 Loïc GRIFFIE. All rights reserved.
+//
+
+import UIKit
+
+/// Class that provide a circular indicator to represent time with customizable options.
+@IBDesignable
+open class MUTime: MUNibView {
+
+    @IBOutlet private var timeLabel: MULabelCounter!
+    @IBOutlet private var timeBackground: UIView!
+
+    /// The progress value text font.
+    @objc open dynamic var timeFont: UIFont = .systemFont(ofSize: 13, weight: .bold) {
+        didSet {
+            timeLabel.textFont = timeFont
+        }
+    }
+
+    /// The progress value text color.
+    @IBInspectable open dynamic var timeColor: UIColor = .clear {
+        didSet {
+            timeLabel.textColor = timeColor
+        }
+    }
+
+    /// The progress value text background color.
+    @IBInspectable open dynamic var timeBackgroundColor: UIColor = .clear {
+        didSet {
+            timeBackground.backgroundColor = timeBackgroundColor
+        }
+    }
+
+    /// Specifies the progress text value format.
+    @IBInspectable open dynamic var format: String = "%.f" {
+        didSet {
+            timeLabel.format = format
+            updateTime()
+        }
+    }
+
+    /// Show / Hide the progress text value. Shown by default.
+    @IBInspectable open dynamic var showValue: Bool = true {
```

Comment:
Maybe hidden the timeLabel will be set the format to ""? Or more readable like that?

Reply:
C'est vrai mais cela permet de faire un bind avec Rx sur un boolean et c'est plus compréhensible je trouve de faire un hidden que de mettre un format à ""